app/elements/submission/tasks/canvas-game-task.html

```diff
@@ -69,6 +69,11 @@
 	</style>
 	<template>
 		<iron-ajax
+			id="getCodeRequest"
+			handle-as="text"
```

Comment:
`handle-as="json"` will spare you the `JSON.parse` in `onTemplateResponse`, just saying.